<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>1081</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>971</width>
        <height>651</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="message_label">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>550</y>
         <width>541</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>消息</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>360</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>上行数量：</string>
        </property>
       </widget>
       <widget class="QLabel" name="up_count_label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>72</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>下行数量：</string>
        </property>
       </widget>
       <widget class="QLabel" name="down_count_label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="com_config_button">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>470</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>串口设置</string>
       </property>
      </widget>
      <widget class="QLabel" name="img_label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>511</width>
         <height>431</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>120</y>
         <width>181</width>
         <height>181</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="strat_detect_Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>开始检测</string>
        </property>
       </widget>
       <widget class="QPushButton" name="open_camera_Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>打开摄像头</string>
        </property>
       </widget>
       <widget class="QLabel" name="start_time_text_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>开始时间：</string>
        </property>
       </widget>
       <widget class="QLabel" name="end_time_text_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>结束时间：</string>
        </property>
       </widget>
       <widget class="QLabel" name="pass_time_text_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>检测时长：</string>
        </property>
       </widget>
       <widget class="QLabel" name="start_time_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
       <widget class="QLabel" name="end_time_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
       <widget class="QLabel" name="pass_time_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>140</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="mavlink_goto_button">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>520</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>mavlink</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>lat:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>lon:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="com_combobox2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>110</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="mavlink_connect_button">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>180</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goback_button">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>250</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>groundspeed:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>alt:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>pitch:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>roll:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>yaw:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ground_speed_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="alt_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lat_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lon_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="pitch_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="roll_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="yaw_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>941</width>
        <height>641</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="uart_tx_data_count_label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>600</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>10</y>
         <width>201</width>
         <height>301</height>
        </rect>
       </property>
       <property name="title">
        <string>串口设置</string>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>51</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>串口号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>停止位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>数据位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>校验位</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>89</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="com_combo_box"/>
         </item>
         <item>
          <widget class="QComboBox" name="baud_combo_box">
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="stopbit_combo_box">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="databit_combo_box">
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="checkbit_combo_box">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Odd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Even</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="uart_en_push_button">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>600</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>发送：</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="uart_rec_show">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>491</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="uart_rx_data_count_label">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>600</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>320</y>
         <width>201</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>接收设置</string>
       </property>
       <widget class="QPushButton" name="save_com">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>保存串口</string>
        </property>
       </widget>
       <widget class="QPushButton" name="uart_clear_rec_push_button">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>清除接收</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="rec_hex_check_box">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>51</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>HEX</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>显示时间</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="page_back_button">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>560</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>600</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>接收：</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="uart_send_show">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>310</y>
         <width>491</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>420</y>
         <width>221</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>发送设置</string>
       </property>
       <widget class="QCheckBox" name="uart_timer_check_box">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>定时发送</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>周期</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="uart_timer_line_edit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="send_hex_check_box">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>51</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>HEX</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_8">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>发送新行</string>
        </property>
       </widget>
       <widget class="QPushButton" name="uart_send_push_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QPushButton" name="uart_send_clear_push_button">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>清除发送</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="message_label2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>550</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>消息</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
